### DISCUSSION TITLE
New beta badges on profile - low if any added value

### DISCUSSION THREAD

**Discussion by MichelML** [community/community#18157] [Other Feature Feedback, Questions, & Ideas]
Not a fan of the new beta profile badges

![image](https://user-images.githubusercontent.com/9539763/172897404-06932a9b-87b0-4460-88cf-ca4e0c2978fe.png)

They seem to me childish, or at least for very beginners, and personally I think it makes my profile page less serious/convincing.

Idea: minimally make this optional (if not possible already)

**Comment by bailey** [comment]
Hi @MichelML! Achievements are totally optional. You can turn them off under `Profile settings` at https://github.com/settings.

**Reply by NS007-dev** [reply]
@MichelML @bailey  I do like the idea of having the badges, because before this point, I thought having badges was rare and that you had to either take part in the archive programs, be a pro, so I thought it was restrictive, but now, they have opened it up a bit more.

Yes, however I do agree that maybe they should be toned down just a bit to make it seem smart and minimal, yet aesthetically pleasing. 

This badge seems nice:
![image](https://user-images.githubusercontent.com/70807500/172898501-0ff9c8b2-9e7e-4c98-92e9-dd370061c5db.png)

However, the others, should be toned a bit down, in my opinion:
![image](https://user-images.githubusercontent.com/70807500/172898600-00bb0248-3765-498a-aa4f-caa0c5052a7e.png)


Thank you.

**Reply by zkldi** [reply]
Turning off achievements also turns off the Arctic Vault and Mars badges though.

**Reply by NS007-dev** [reply]
I mean I do really like them, but for those who are doing GitHub professionally, it may not appeal to them. Like @bailey said, it could just be switched off in the settings. Thanks.

**Reply by bailey** [reply]
@zkldi We're planning to ship more granular controls in the future so you can choose which achievements to surface publicly. Thank you for this feedback!
